.changeset/fresh-carrots-fly.md

```diff
@@ -0,0 +1,7 @@
+---
+'@primer/react-brand': minor
+---
+
+⚠️ Breaking visual change:
+
+`MinimalFooter` now respects the ordering of the `socialLinks` prop when rendering social links, e.g. `<MinimalFooter socialLinks={['x', 'tiktok', 'youtube']} />` will render the links in that order.
```

Comment:
This reads more severe than is likely necessary. Definitely good to call it out, but I think we should use the term 'breaking' sparingly as it didn't even set our own VRT off due to it being more feature-add than feature-removal. Could we please revise the changeset to say something like:

```suggestion
`MinimalFooter` now respects the ordering of the `socialLinks` prop when rendering social links, e.g. `<MinimalFooter socialLinks={['x', 'tiktok', 'youtube']} />` will render the links in that order. 

Note: This may constitute a visual breaking change if you were relying on the social link order to _not_ be respected.
```